xml/System.Net.Http/HttpClient.xml

```diff
@@ -455,7 +457,7 @@ The environment variables used for `DefaultProxy` initialization on Windows and
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Headers set on this property don't need to be set on request messages again.  
+ Headers set on this property don't need to be set on request messages again. `DefaultRequestHeaders` should not be modified after any request has been sent, because it is not thread-safe.
```

Comment:
Do we still need the first sentence? It reads like: you don't need to reset the headers, also do not reset the headers at all.